### DISCUSSION TITLE
使用默认播放器模式但是页面会下移

### DISCUSSION THREAD

**Discussion by tomto0810** [the1812/Bilibili-Evolved#2379] [综合讨论 / General]
本来觉得应该算问题，想了想还是投建议吧

**关于哪一项功能**
设置/视频/使用默认播放器模式/默认播放器模式：宽屏

**问题描述**

设置为宽屏，并且取消播放时应用，每次进页面会自动宽屏但是页面会下移，需要手动去滚轮滚回去

**脚本版本**

Stable v1.12.19

**浏览器版本**

火狐浏览器
版本: 92.0.1 (64 位)

**播放器版本**

2.68.4-40ae5944

**Comment by Geeyun-JY3** [comment]
自信点，这就是问题。

是不是还开了 `自动定位到播放器` 或 `播放器置顶`：#483
